src/lib/daccCentreValDeLoireExpe.js

```diff
@@ -64,22 +68,43 @@ export const sendCentreValDeLoireMeasuresToDACC = async (client, account) => {
     return null
   }
   logService('info', `Compute measures for ${timeseries.length} timeseries`)
-  const measures = await computeRawMeasures(timeseries)
-  if (!measures) {
+  const measures = computeRawMeasures(timeseries)
+  if (!measures || (!measures.sectionMeasures && !measures.tripMeasures)) {
     logService('info', 'No measure to send')
     return
   }
-  logService(
-    'info',
-    `Build ${
-      Object.keys(measures).length
-    } measures with user type:  ${userType}`
-  )
-
-  const daccMeasures = await buildDACCMeasures({
-    measures,
-    userType
-  })
+  const daccMeasures = []
+  if (measures.tripMeasures && Object.keys(measures.tripMeasures).length > 0) {
+    logService(
+      'info',
+      `Build ${
+        Object.keys(measures.tripMeasures).length
+      } measures with user type:  ${userType}`
+    )
+    const daccMeasures = await buildDACCMeasures({
+      measures: measures.tripMeasures,
+      userType,
+      isSectionLevel: false
+    })
+    daccMeasures.push(...daccMeasures)
```

Comment:
Careful you are redeclaring `const = daccMeasures`. I do not know in which `daccMeasures` values are pushed.

Reply:
Oh very good catch, thank you

Reply:
Fixed now